src/ipv4.c

```diff
@@ -54,6 +57,15 @@ static void ipv4_default_ttl_handler(vector_t tokens)
     FREE_PTR(str);
 }
 
+static void ipv4_forward_handler(vector_t tokens)
+{
+    char *str = set_value(tokens);
+    assert(str);
+    if (!strncmp(str, "on", strlen("on")) || !strncmp(str, "ON", strlen("ON")))
```

Comment:
You can use `strcasecmp`, and do not use 'n' version here, or the value "onABC" will also match.